content/200-concepts/050-overview/100-what-is-prisma/index.mdx

```diff
@@ -253,13 +246,6 @@ Note that when using TypeScript, the result of this query will be _statically ty
 
 As mentioned above, there are two ways for "getting" your data model into the Prisma schema. Depending on which approach you choose, your main Prisma workflow might look different.
 
-<Admonition type="warning">
-
-**MongoDB not supported**<br />
-Introspection and Prisma Migrate do not yet support the MongoDB connector. To use MongoDB while it is in Preview, you must design your Prisma schema to match the shape of your data in MongoDB.
-
```

Comment:
Both Migrations and Introspection as they are explained and mentioend below work for MongoDB.